pkg/util/proxy.go

```diff
@@ -161,3 +162,15 @@ func ValidateProxyURL(proxyUrl string) error {
 	}
 	return nil
 }
+
+// parseProxySpecialChar parse the URI and convert special char to hex
+func parseProxySpecialChar(uri string) string {
+	u, _ := url.Parse(uri)
+	scheme := fmt.Sprintf("%s://", u.Scheme)
+	if strings.HasPrefix(uri, scheme) {
+		s := strings.Replace(uri, scheme, "", 1)
+		s = url.PathEscape(s)
```

Comment:
:+1:

Reply:
👍

Reply:
If we can get confirmation that this works for the person who created the issue then we can merge it.

Reply:
Have we pointed the artifact binary to the user who created this issue? @praveenkumar @anjannath